lnrpc/rpc.proto

```diff
@@ -2768,3 +2783,19 @@ message BakeMacaroonResponse {
     /// The hex encoded macaroon, serialized in binary format.
     string macaroon = 1 [json_name = "macaroon"];
 }
+
+message ChannelRevenueRequest{
+    ///  The outpoint (txid:index) of the funding transaction.
+    ChannelPoint channel_point = 1;
+
+    /// Start time is the beginning of the range that revenue should be calcualted over.
+    uint64 start_time = 2 [json_name = "start_time"];
```

Comment:
Add something to the field name to clarify the units. Is it unix time?

Reply:
In the comment or var name? `start_time_seconds`/ `start_time_unix` would be most appropriate in the name, but we don't seem to have that elsewhere in the rpc for times. Otherwise I'll just update the comment.

Reply:
That is true indeed. Recently we got `time_ns`, but that still doesn't express that it is unix time. Ok, comment then indeed.